fedimint-cli/Cargo.toml

```diff
@@ -13,7 +13,6 @@ description = "Fedimint client CLI interface"
 rustc-args = ["--cfg", "tokio_unstable"]
 
 [features]
-default = ["tor"]
```

Comment:
No one should need to import `fedimint-cli` as a dependency, so this one can enable tor by default and we probably want to ship `fedimint-cli` with tor support.

Reply:
Fixed